src/main/java/picocli/CommandLine.java

```diff
@@ -1929,10 +1929,30 @@ public int execute(String... args) {
         }
     }
     private static int handleUnhandled(Exception ex, CommandLine cmd, int defaultExitCode) {
-        ex.printStackTrace(cmd.getErr());
+        cmd.getErr().print(exceptionToColorString(ex, cmd.getColorScheme()));
+        cmd.getErr().flush();
```

Comment:
Is this call to `flush()` necessary?

Reply:
I added this line to fix a bug I find during testing. It seems that Java's `PrintWriter` won't flush until a line separator is met, so I'd like to add one `flush()` here to make sure the message is indeed printed. In retrospect, this line might not be necessary, as error messages generated with `exceptionToColorString` should already have line separators in it. But... just to be safe?